package.json

```diff
@@ -41,6 +41,7 @@
     "mocha-lcov-reporter": "^1.2.0",
     "mock-spawn": "^0.2.1",
     "mockery": "~1.4.0",
+    "nightwatch-component-a": "git://github.com/cesine/nightwatch-component-a",
```

Comment:
Not sure if it's a goo idea to rely on this external package.
It would make more sense to either have a test packing in nightwatchjs repo, or better still,
simply use and this package into an examples or packages/test etc.

Would moving it internally also be a fair example?